CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+# Contents
+
+This document describes in detail the contents of this repository. It is still a *work in progress*,
+but do not hesitate to report if something is missing.
+
+## Overview
+
+The repository contains a set of tools to aid in the process of building, testing and releasing
+(open)SUSE based distributions. The [Tools](#tools) section enumerates and describes all these
+tools, including links to documentation, source code and some information about where they are used.
+
+Apart from these tools, the repository includes:
+
+* Some documenation in the [docs](docs) directory.
+* A Python module called [osclib](osclib) which includes code that is shared by several tools. They
+  are available in the `osclib` package.
+* A Docker-based [tests suite](tests). The Docker manifests and the Docker Compose files are
+  located in the [dist](dist) directory.
+* [GoCD](https://www.gocd.org) configuration files in [gocd](gocd). GoCD is an open source CI/CD
+  server that it is used to deploy the bots on OBS.
+* A set of [Tampermonkey](https://www.tampermonkey.net) scripts (see [userscript](userscript)
+  directory) to extend OBS features when using the web interface.
+* Several [systemd](systemd) units, as many of the tools (e.g., the `staging-bot`) make use of
+  them.
+
+## Tools
+
+Most of these tools are available as packages for several distributions. Check the [spec file in
+this repository](dist/package/openSUSE-release-tools.spec) or the [devel
+project](https://build.opensuse.org/package/show/openSUSE:Tools/openSUSE-release-tools) for further
+information.
+
+For the time being, we have classified them into three different groups: *command line tools*, *OBS
+bots* and *osc plugins*. Bear in mind that the information in the following list might be wrong and
+incomplete.
+
+### Command Line Tools
+
+ Usually, the executables are renamed as `osrt-NAME` (e.g., `osrt-announcer`).
+
+#### announcer
+
+Generates email diffs summaries to announce product releases.
+
+* Sources: [factory-package-news/announcer.py](factory-package-news/announcer.py)
+* Documentation: [factory-package-news/README.asciidoc](factory-package-news/README.asciidoc)
+* Package: openSUSE-release-tools-announcer
```

Comment:
I would say usage is opensuse-factory mailing list?

Reply:
I am not sure where and how it is deployed

Reply:
gocd/announcer.gocd.yaml - the announcer package is no longer relevant